python_modules/dagster/dagster/_core/definitions/asset_subset.py

```diff
@@ -85,6 +85,13 @@ def size(self) -> int:
         else:
             return len(self.subset_value)
 
+    @property
+    def is_empty(self) -> bool:
```

Comment:
No change necessary because this problem predates this PR but this inspired https://github.com/dagster-io/internal/discussions/9541 :-)